lib/registries/addon/branded/discover/template.hbs

```diff
@@ -0,0 +1,19 @@
+{{page-title (t 'registries.discover.page_title') prepend=false}}
+<RegistriesNavbar
+    class='RegistriesStyle'
+    @campaign='{{if @provider @provider.id 'osf'}}-registries'
+    @provider={{this.model}}
+    {{with-branding (get-model this.model.brand)}}
+/>
```

Comment:
I think we can replace this with `<RegistriesWrapper>` as that handles the navbar and the other banner stuff too.

Reply:
It looks like we had `data-analytics-scope='Registries Discover page'` on the RegistriesWrapper before too. (And this reminds me to add a `data-analytics-scope` to the preprints discover page too 😬 )